applications/openshift/api-server/api_server_api_priority_gate_enabled/rule.yml

```diff
@@ -0,0 +1,44 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: 'Enable the APIPriorityAndFairness feature gate'
+
+description: |-
+    To limit the rate at which the API Server accepts requests, make
+    sure that the API Priority and Fairness feature is enabled.
```

Comment:
Need to add how it is enabled in the description.

Reply:
added